Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN npm ci --verbose --no-optional && npm cache clean --force
 
 COPY . .
 
+RUN npx mesh build
```

Comment:
this is currently being run in the `entrypoint.sh`: https://github.com/chanzuckerberg/czid-graphql-federation-server/blob/9af53fe5d9b8c0f88659014e054c430f708e1cfd/entrypoint.sh#L2

[`npm run build` script defined in package.json](https://github.com/chanzuckerberg/czid-graphql-federation-server/blob/9af53fe5d9b8c0f88659014e054c430f708e1cfd/package.json#L7).  

Is there any reason we want to run build a separate time here?

I think we do want a single command to run build and start (which is why they're in a script together), to make sure the server is built with the latest schema.

Reply:
This can be removed I guess. I added it there because I was running into an issue where it complained that the assets weren't already built. This happened after I had deleted the build folder

Reply:
bump, can this be removed without issue?